### ISSUE TITLE
Circuit Of the month/feature.

### ISSUE THREAD

**Issue by Mayank77maruti** [CircuitVerse/CircuitVerse#4480]
### Is your feature request related to a problem? Please describe

This feature request is not related to a problem rather it is a feature that would keep the users attention on the website intact for a longer time and would help them to learn about new circuits.

### Describe the solution you'd like

I would like to make a new section namly circuit of the month / year that would be based on suitable parameters like the most upvoted circuit and so on. In this section we would feature one circuit and all the rest functionalities would be same where the users can click on and explore more about the circuit.

### Are you working on this?

Yes

**Comment by salmoneatenbybear** [comment]
Cc : @Asrani-Aman probable ideation conflict with weekly contest.

**Comment by Asrani-Aman** [comment]
Yeah that's the exact reason why we are making weekly contest feature

**Comment by salmoneatenbybear** [comment]
Closing as scope overlaps other decided efforts.
